integrationTests/vm/arwen/gasScheduleV2.toml

```diff
@@ -0,0 +1,607 @@
+[BuiltInCost]
```

Comment:
Suggestion to not have this file but instead link it to .../elrond-go/node/cmd/config/gasSchedules as to avoid the situation when we push modifications only in the node config file without changing this file

Reply:
First we change this file. Only after the one on node side. In the past it did not work to link from config. Will try it now.

Reply:
It will work and will comply with the rest of the project when the align factories to specs will be merged (that already uses the original files from node/cmd/config).